site/de/_posts/2021-01-19-phase3.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+lang: de
+lang-ref: DFG Bewilligung
+title: "Phase III der OCR-D-Koordinierung bewilligt"
+tags: [DFG, grant, phase 3]
+author: Elisabeth Engl
+---
+
+Der Antrag des Koordinierungsprojekts für die dritte Phase der OCR-D-Förderinitiative wurde von der DFG bewilligt.
+In Phase III werden die Ergebnisse der vorangegangenen Modulprojekt-Phase stabilisiert und der produktive Einsatz der
+OCR-D-Software in der Massendigitalisierung technisch und organisatorisch eingeleitet.
+
```

Comment:
Der Antrag des Koordinierungsprojekts für die dritte Phase der OCR-D-Förderinitiative wurde im Januar 2021 von der DFG bewilligt. In Phase III werden die Ergebnisse der vorangegangenen Modulprojekt-Phase optimiert und der produktive Einsatz der OCR-D-Software in der Massendigitalisierung technisch und organisatorisch eingeleitet.